packages/core/src/extensions/Blocks/nodes/Block.module.css

```diff
@@ -27,11 +27,11 @@ NESTED BLOCKS
   margin-left: 1.5em;
 }
 
-.blockGroup .blockGroup > .blockOuter {
```

Comment:
I think it's better to keep the formatting as it was. Admittedly the CSS isn't very readable at the moment, but it would make it much easier to keep track of the more important changes in the PR.